extensions/hints/init.lua

```diff
@@ -37,6 +37,22 @@ hints.fontSize = 0.0
 --- The default is 4. Setting to 0 will disable this feature.
 hints.showTitleThresh = 4
 
+--- hs.hints.titleMaxSize
+--- Variable
+--- If the title is longer than maxSize, the string is truncated, -1 to disable, valid value is >= 6
+hints.titleMaxSize = -1
+
+--- hs.hints.titleRegex
+--- Variable
+--- List of regex substituions for window titles, in string.gsub style format:
+---    { key = { pat="<gsub_pattern>", sub="<gsub_replace>" }, ... }
+---
+---    Add as many keys as you like
+---
+--- e.g. { MSApps={pat="Microsoft%s*", sub=""}, Chrome={pat="Google Chrome", sub="Chrome"} }
+--- Only valid for vimperator style
+hints.titleRegexSub = {}
```

Comment:
This should be made a list/array/integer table, there is no reason to have the keys here. Having them is confusing since it looks like they have meaning when in fact they have none.